pkg/db/drivers/etcd/etcd.go

```diff
@@ -1494,12 +1510,12 @@ func (c *Client) getProfileByName(ctx *c.Context, name, storageType string) (*mo
 
 // GetDefaultProfile
 func (c *Client) GetDefaultProfile(ctx *c.Context) (*model.ProfileSpec, error) {
-	return c.getProfileByName(ctx, "default", "block")
+	return c.getProfileByNameAndType(ctx, "default", "block")
```

Comment:
Now the default profile created will be either default_block or default_file, so this call will always return nil. Right? Should we use default_block here

Reply:
You are right.
I have changed this default name from "default" to "default_block"(same as the default name in install.sh)